src/pages/news/news-detail/news-detail.js

```diff
@@ -115,4 +118,14 @@ const NewsDetailPage = ({ match }) => {
   );
 };
 
+const newsDetailPropTypes = {
+  params: PropTypes.shape({
+    id: PropTypes.string
+  })
+};
+
+NewsDetailPage.propTypes = newsDetailPropTypes;
+
+NewsDetailPage.defaultProps = newsDetailPropTypes;
```

Comment:
`defaultProps` contains actual default values for the props

Reply:
I made isRequired and we don't need default any more